exercises.py

```diff
@@ -0,0 +1,296 @@
+'spam' * 3 = SPAMSPAMSPAM 
+
+8. Why is eggs a valid variable name while 100 is invalid? = 100 SHOULD BE DEFINED BY str(100) A VARIABLE CANT START WITH A NUMBER.
+
+9. What three functions can be used to get the integer, floating-point number, or string version of a value? = INT(), FLOAT(), STR()
+
+10. Why does this expression cause an error? How can you fix it?
+'I have eaten ' + 99 + ' burritos.' = 99 IS AN INTEGER. 99 SHOULD BE DEFINED AS str(99).
+
+# Chapter 2
+1. What are the two values of the Boolean data type? How do you write them? = TRUE AND FALSE; Written, True/False
+
+2. What are the three Boolean operators? AND, OR, NOT
+
+3. Write out the truth tables of each Boolean operator (that is, every possible combination of Boolean values for the operator and what they evaluate to).
+   TRUE and TRUE = True
+   TRUE and FALSE = False
+   FALSE and TRUE = False
+   FALSE and FALSE = False
+   TRUE or TRUE = True
+   TRUE or FALSE = True
+   FALSE or TRUE = True
+   FALSE or FALSE = False
+   not TRUE = False
+   not FALSE = True
+
+4. What do the following expressions evaluate to?
+(5 > 4) and (3 == 5)
+not (5 > 4) 
+(5 > 4) or (3 == 5)
+not ((5 > 4) or (3 == 5))
+(True and True) and (True == False)
+(not False) or (not True)
+False
+False
+True
+False
+False
+True
+
+5. What are the six comparison operators? ==, !=, <, >, <=, >=
+
+6. What is the difference between the equal to operator and the assignment operator?
+The == operator asks is two values are the same as each other
+The = The = operator puts the value on the right into the variable on the left.
+
+7. Explain what a condition is and where you would use one.
+A condition is used in a flow control statement and gives a True or False value.
+
+8. Identify the three blocks in this code:
+spam = 0
+if spam == 10:
+    print('eggs')
+    if spam > 5:
+        print('bacon')
+    else:
+        print('ham')
+    print('spam')
+print('spam')
+PRINT('EGGS')
+IF SPAM > 5:
+PRINT(BACON)
+ELSE:
+PRINT('HAM')
+PRINT('SPAM')
+
+9. Write code that prints Hello if 1 is stored in spam, prints Howdy if 2 is stored in spam, and prints Greetings! if anything else is stored in spam.
+if spam == 1:
+    print('Hello')
+elif spam == 2:
+    print('Howdy')
+else: 
+    print('Greetings')    
+
+10. What keys can you press if your program is stuck in an infinite loop?
+Ctrl-C
+
+11. What is the difference between break and continue?
+A BREAK breaks out of a while loop early/A CONTINUE continues the while loop
+
+12. What is the difference between range(10), range(0, 10), and range(0, 10, 1) in a for loop?
+RANGE(10) - SETS RANGE 0 UP TO 10
+RANGE(0, 10) - SETS RANGE SPECIFICALLY FROM 0-10
+RANGE(0, 10, 1) - INCREASES VARIABLE BY 1 ON EACH LOOP
+
+13. Write a short program that prints the numbers 1 to 10 using a for loop. Then write an equivalent program that prints the numbers 1 to 10 using a while loop.
+for i in range(1 11):
+print(i)
+
+i = 1
+while i <= 10
+print(i)
+
+i = i + 1
```

Comment:
This will work if you fix the indentations and colons. line 122 runs one more time than it needs to. Can you think about how you could refactor the code to avoid it?